hw/hw3/task3.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+class Users {
+  constructor(numberOfUsers) {
+    this.numberOfUsers = numberOfUsers;
+  }
+
+  async getUsers() {
+    let url = `https://randomuser.me/api/?results=${this.numberOfUsers}`;
+    let usersData = await this.makeGetRequest(url);
+    let users = "";
+    for (let key in usersData["results"]) {
```

Comment:
Better to use reduce instead of `for in`

Reply:
fixed